curriculum/challenges/english/blocks/lecture-html-fundamentals/670803abcb3e980233da4768.md

```diff
@@ -35,7 +34,10 @@ Add another `section` element below the first one. Then inside of the `section`
 ```html
 <section>
   <h2>Mammals</h2>
-  <p>Mammals are warm-blooded animals with fur or hair. Most give birth to live young.</p>
+  <p>
+    Mammals are warm-blooded animals with fur or hair. Most give birth to live
+    young.
+  </p>
```

Comment:
The formatting changes were probably caused by a misconfig, which #64054 will adress.

I think the whitespace changes can stay, but let's revert this one.

```suggestion
  <p>Mammals are warm-blooded animals with fur or hair. Most give birth to live young.</p>
```